pilot/pkg/model/extensions.go

```diff
@@ -27,9 +27,12 @@ import (
 	"google.golang.org/protobuf/types/known/wrapperspb"
```

Comment:
we should be adding validation to pkg/config/validation/validation.go for the new fields